tests/configurable-timeout-test.ts

```diff
@@ -36,11 +36,11 @@ QUnit.test('We can use a custom later', function(assert) {
     }
   });
 
-  bb.scheduleOnce('one', () => {
+  bb.later(() => {
```

Comment:
I should either do
1. use custom `next` `clearNext` with `scheduleOnce`
2. use custom `setTimeout ` `clearTimeout ` with `later`
or your thoughts